test.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 var assert = require('assert');
 var chalk = require('./');
+var requireUncached = require('require-uncached');
+var resolveFrom = require('resolve-from');
+var semver = require('semver');
```

Comment:
Can you move these up before `var chalk = require('./');` and after `var assert = require('assert');`?